content/docs/quickstart/go.md

```diff
@@ -3,122 +3,54 @@ title: Go Quick Start
-        your operating system from [github.com/google/protobuf/releases][]
-        (`protoc-<version>-<os><arch>.zip`). For example:
+- **[Protocol buffer][pb] compiler**, `protoc`, [version 3][proto3].
 
-        ```sh
-        $ PB_REL="https://github.com/protocolbuffers/protobuf/releases"
-        $ curl -LO $PB_REL/download/v{{< param protoc-version >}}/protoc-{{< param protoc-version >}}-linux-x86_64.zip
-        ```
+  For installation instructions, see [Protocol Buffer Compiler
+  Installation][pbc-install].
 
-     2. Unzip the file under `$HOME/.local` or a directory of your choice. For
-        example:
+- **Go plugin** for the protocol compiler:
 
-        ```sh
-        $ unzip protoc-{{< param protoc-version >}}-linux-x86_64.zip -d $HOME/.local
-        ```
+   1. Install the protocol compiler plugin for Go (`protoc-gen-go`) using the
+      following command:
 
-     3. Update your environment's path variable to include the path to the
-        `protoc` executable. For example:
+      ```sh
+      $ go get github.com/golang/protobuf/protoc-gen-go@v1.3.5
+      ```
 
-        ```sh
-        $ export PATH="$PATH:$HOME/.local/bin"
-        ```
+   2. Update your `PATH` so that the `protoc` compiler can find the plugin:
 
-    > **MacOS note**: Using [Homebrew][]? Simply run: `brew install protobuf`.
-
- 2. The `protoc` plugin for Go (`protoc-gen-go`) was installed as a dependency
-    of the `grpc` module. You can confirm this, or install the plugin, using the
-    following command:
-
-    ```sh
-    $ go get github.com/golang/protobuf/protoc-gen-go
-    ```
-
- 3. Update your `PATH` so that the `protoc` compiler can find the plugin:
-
-    ```sh
-    $ export PATH="$PATH:$(go env GOPATH)/bin"
-    ```
+      ```sh
+      $ export PATH="$PATH:$(go env GOPATH)/bin"
+      ```
 
-### Copy the example
+### Get the example code
 
-The example code is part of the `grpc` source, which you fetched by following
-steps of the previous section. You'll need a local copy of the example code to
-work through this quick start.
-
- 1. Choose a suitable working directory, and ensure that it exists:
-
-     ```sh
-    $ export MY_EXAMPLES="$HOME/examples"
-    $ mkdir -p "$MY_EXAMPLES"
-    ```
-
- 2. Copy the example source:
-
-    ```sh
-    $ EX_SRC_DIR="$(go env GOPATH)/pkg/mod/google.golang.org/grpc@{{< param grpc_release_tag >}}/examples"
-    $ cp -R "$EX_SRC_DIR/helloworld" "$MY_EXAMPLES"
-    ```
-
- 3. Ensure that the example files are writable since you'll be making changes soon:
-
-    ```sh
-    $ chmod -R u+w "$MY_EXAMPLES/helloworld"
-    ```
-
- 4. Change to the example's directory:
-
-    ```sh
-    $ cd "$MY_EXAMPLES/helloworld"
-    ```
+The example code is part of the [grpc-go][] repo.
 
- 5. Setup the example as a module:
+ 1. [Dowload the repo as a zip file][download] and unzip it, or clone
+    the repo:
 
     ```sh
-    $ go mod init examples/helloworld
+    $ git clone https://github.com/grpc/grpc-go
```

Comment:
@dfawley - in the end, it turns out to be easier to just clone the repo to get the sources. Having the example as a separate module still helps though!